packages/discovery-styles/scss/components/document-preview/_document-preview-pdf-viewer.scss

```diff
@@ -12,3 +12,16 @@
 .#{$prefix}--document-preview-pdf-viewer--text {
   transform-origin: left top 0px;
 }
+
+.#{$prefix}--document-preview-pdf-viewer-highlight {
+  position: absolute;
+  transform-origin: left top 0px;
+  top: 0;
+  left: 0;
+}
+
+.#{$prefix}--document-preview-pdf-viewer-highlight--item {
+  position: absolute;
+  opacity: 0.5;
+  background: rgba(0, 0, 255, 1);
```

Comment:
Same here: need to check with Design about use of highlight color/opacity.

Reply:
OK. Use the Carbon color with opacity as a default and check with design.